force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +50,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+       
+        if(age>=18){
+            system.debug(age);
+            return true;
+        }else{
+            system.debug('If age<18 the not eligible to vote');
+            return false;
+        }
+         // Replace null with the variable you used to store the result
```

Comment:
Good, think about how you can solve this with out a conditional. These sorts of problems will show up every day on the job. What do expressions using < or > evaluate to?